games/data/lethal-company.yml

```diff
@@ -2,7 +2,7 @@ uuid: "40c7d846-cff1-45fe-8679-4abe993abafb"
 label: "lethal-company"
 meta:
   displayName: "Lethal Company"
-  iconUrl: "None"
+  iconUrl: "https://styles.redditmedia.com/t5_9al6ft/styles/communityIcon_lb87s4o45y0c1.png"
```

Comment:
Hey, this field doesn't actually do anything yet & should be left empty for now given that we don't know how exactly will it work yet. It's in the schema mostly as a stub at this stage.